data/points/human2.txt

```diff
@@ -0,0 +1,4706 @@
+0.144529 -0.014328 0.381773
```

Comment:
What is the purpose of adding human2.txt file ? It is the same as human.off (except the header part), and it is not used in code, nor test.